tax/tag.go

```diff
@@ -0,0 +1,27 @@
+package tax
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/validation"
+)
+
+// Tag describes a tax tag that can be used to identify additional
+// requirements in an electronic invoice.
+type Tag struct {
+	// Key used to identify the tag
+	Key cbc.Key `json:"key" jsonschema:"title=Key"`
+	// Name of this tag.
+	Name i18n.String `json:"name,omitempty" jsonschema:"title=Name"`
+	// Additional local
+	Meta cbc.Meta `json:"meta,omitempty" jsonschema:"title=Meta"`
```

Comment:
I liked how schemes had Key, Name, and Description. Perhaps tags can take a similar approach. Might be useful for UI? 

(I'd be happy to take that on once this gets merged!)

Reply:
Description could be nice! Because there are quite a few tags, I felt it was easier to have slightly longer names. Happy to refactor later though!